utils/oscap-xccdf.c

```diff
@@ -443,6 +447,39 @@ static int callback_scr_result_progress(struct xccdf_rule_result *rule_result, v
 	return 0;
 }
 
+static int callback_scr_rule_progress_full(struct xccdf_rule *rule, void *arg)
+{
+	const char * rule_id = xccdf_rule_get_id(rule);
+
+	if (!xccdf_policy_is_item_selected((struct xccdf_policy *)arg, rule_id))
+		return 0;
+
+	const char *title = xccdf_policy_get_readable_item_title((struct xccdf_policy *)arg, (struct xccdf_item *)rule, NULL);
+	printf("%s|%s|", rule_id, title);
+	free((char *)title);
```

Comment:
This shouldn't be `const` if you modify it by `free()`. The function `xccdf_policy_get_readable_item_title` returns just `char *`

Reply:
Fixed.